### PR TITLE
feature: add the yamllint tool

### PR DESCRIPTION
**Pull request by fengzixu**
Signed-off-by: fengzixu <hnustphoenix@gmail.com>

### CODE REVIEW

**Comment by fengzixu** [comment]
@allencloud Request review

File: Dockerfile [hunk 1]
```diff
@@ -51,3 +51,9 @@ RUN chmod +x /usr/local/bin/swagger
 
 # install misspell
 RUN curl -L https://git.io/misspell | bash
+
+# install yamllint
+RUN apt-get install -y python-pip
```
Comment (allencloud) [inline Dockerfile]: Please move this line to L14, and combine the following 3 lines into a single one.
We can reduce the image size by that.

**Comment by fengzixu** [comment]
Done @allencloud

**Comment by allencloud** [comment]
LGTM

**Pull request merged**